src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -418,6 +419,11 @@ private void HandleChangesForStaticIndexes(DatabaseRecord record, long index, Li
                 var name = kvp.Key;
                 var definition = kvp.Value;
 
+                if (definition.Name.StartsWith(Constants.Documents.Indexing.SideBySideIndexNamePrefix, StringComparison.OrdinalIgnoreCase))
+                {
+                    throw new RachisInvalidOperationException($"Index name cannot start with {Constants.Documents.Indexing.SideBySideIndexNamePrefix} but got {definition.Name} of index {name}");
+                }
+
```

Comment:
let's also move this exception to be under the `try/catch` and change the exception type to `InvalidOperationException`

Reply:
Done